CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
-## 0.6.5-dev0
+## 0.6.5
 
 * Fix `source` property for elements generated by pdfminer.
 * Add 'OCR-tesseract' and 'OCR-paddle' as sources for elements generated by OCR.
 
 ## 0.6.4
 
-* Add functionality to keep extracted image elements while merging inferred layout with extracted layout
```

Comment:
this line should move to `0.6.5` ?

Reply:
`imo we should just cut the 0.6.5 release now rather than accumulate more potentially breaking changes in unstructured-inference`

IMO, the line should move to 0.6.6?

Reply:
moved the line to `0.6.5`